src/assets/data/localisations.ts

```diff
@@ -276,354 +351,467 @@ export const Localisations = {
+    pl: 'Rozmiar',
+    hb: 'גודל'
   },
   SETTINGS_LAST_UPLOAD_TO_SERVER: {
     da: 'Sidste upload til server',
     de: 'Letzter Upload zum Server',
     en: 'Last Upload to Server',
     es: 'Última carga al servidor',
     it: 'Ultimo caricamento sul server',
-    nl: 'Laatste upload naar server'
+    nl: 'Laatste upload naar server',
+    pl: 'Ostatnie wysyłanie na serwer',
+    hb: 'העלאה אחרונה לשרת'
   },
   SETTINGS_DEBUGGING: {
     da: 'Debugging',
     de: 'Debuggen',
     en: 'Debugging',
     es: 'Depuración',
     it: 'Debug',
-    nl: 'Debugging'
+    nl: 'Debugging',
+    pl: 'Debugowanie',
+    hb: 'מחפש באגים'
   },
   SETTINGS_GENERATE_NOTIFS: {
     da: 'Generer Testmeddelelse',
     de: 'Testbenachrichtigung Generieren',
     en: 'Generate Test Notifications',
     es: 'Generar Notificación de Prueba',
     it: 'Genera Notifica di Prova',
-    nl: 'Genereer Testmelding'
+    nl: 'Genereer Testmelding',
+    pl: 'wygeneruj powiadomienie o badaniu',
+    hb: 'הפק התראות מבחן '
   },
   SETTINGS_LOG_NOTIFS: {
     da: 'Logmeddelelser',
     de: 'Protokollbenachrichtigungen',
     en: 'Log Notifications',
     es: 'Notificaciones de Registro',
     it: 'Registra le Notifiche',
-    nl: 'Logmeldingen'
+    nl: 'Logmeldingen',
+    pl: 'Powiadomienia z dziennika',
+    hb: 'נעל התראות'
   },
   SETTINGS_SEND_CACHED_DATA: {
     da: 'Send Cachelagrede Data',
     de: 'Senden Sie Zwischengespeicherte Daten',
     en: 'Send Cached Data',
     es: 'Enviar Datos en Caché',
     it: 'Invia Dati Memorizzati nella Cache',
-    nl: 'Gegevens in Cache Verzenden'
+    nl: 'Gegevens in Cache Verzenden',
+    pl: 'Wyślij dane z pamięci podręcznej',
+    hb: 'שלח נתונים במטמון'
   },
   LANGUAGE_ENGLISH: {
     da: 'Engelsk',
     de: 'Englisch',
     en: 'English',
     es: 'Inglés',
     it: 'Inglese',
-    nl: 'Engels'
+    nl: 'Engels',
+    pl: 'Angielski',
+    hb: 'אנגלית'
   },
   LANGUAGE_SPANISH: {
     da: 'Spansk',
     de: 'Spanisch',
     en: 'Spanish',
     es: 'Español',
     it: 'Spagnolo',
-    nl: 'Spaans'
+    nl: 'Spaans',
+    pl: 'Hiszpański',
+    hb: 'ספרדית'
   },
   LANGUAGE_ITALIAN: {
     da: 'Italiensk',
     de: 'Italienisch',
     en: 'Italian',
     es: 'Italiano',
     it: 'Italiano',
-    nl: 'Italiaans'
+    nl: 'Italiaans',
+    pl: 'Włoski',
+    hb: 'איטלקית'
   },
   LANGUAGE_GERMAN: {
     da: 'Tysk',
     de: 'Deutsch',
     en: 'German',
     es: 'Alemán',
     it: 'Tedesco',
-    nl: 'Duits'
+    nl: 'Duits',
+    pl: 'Nimiecki',
+    hb: 'גרמנית'
   },
   LANGUAGE_DANISH: {
     da: 'Dansk',
     de: 'Dänisch',
     en: 'Danish',
     es: 'Danés',
     it: 'Danese',
-    nl: 'Deens'
+    nl: 'Deens',
+    pl: 'Duński',
+    hb: 'דנית'
   },
   LANGUAGE_DUTCH: {
     da: 'Hollandsk',
     de: 'Niederländisch',
     en: 'Dutch',
     es: 'Holandés',
     it: 'Olandese',
-    nl: 'Nederlands'
+    nl: 'Nederlands',
+    pl: 'Holenderski (niderlandzki)',
+    hb: 'הולנדית'
+  },
+  LANGUAGE_POLISH: {
```

Comment:
@JeroenDMW This still needs to be changed.

Reply:
added in my fork